ui/ducks/app/app.test.js

```diff
@@ -352,22 +352,4 @@ describe('App State', () => {
 
     expect(state.isMouseUser).toStrictEqual(true);
   });
-
-  it('sets gas loading', () => {
```

Comment:
These tests are removed versus being moved because they will eventually be obsolete in #10965